### PR TITLE
changes on validation page

### PR DESCRIPTION
**Pull request by AmrNabel**

### CODE REVIEW

File: src/components/Validate/index.tsx [hunk 1]
```diff
@@ -66,8 +67,7 @@ const Validate = ({ toggleSnackbar, setSnackbarMessage }: IHomeProps) => {
         setStatementValue(res.data.statement)
         setObjectValue(res.data.object)
         setAmtValue(res.data.amt)
-        const dayPart = res.data.effectiveDate.split('T')[0] || res.data.effectiveDate
-        setEffectiveDateValue(dayPart)
+        setEffectiveDateValue(res.data.effectiveDate)
```
Comment (gvelez17) [inline src/components/Validate/index.tsx:70]: no, please keep the split, just check first if effectiveDate is defined - i just checked this in on live server

File: src/components/Validate/index.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import Box from '@mui/material/Box'
 import { useNavigate } from 'react-router-dom'
 import Typography from '@mui/material/Typography'
-import { TextField, Button, MenuItem, FormControl, InputLabel, OutlinedInput, InputAdornment } from '@mui/material'
```
Comment (gvelez17) [inline src/components/Validate/index.tsx:4]: why did you remove these i just added them?

Comment (AmrNabel) [reply src/components/Validate/index.tsx:4]: I replaced the FormControl with the text field
